src/sonic-bgpcfgd/bgpcfgd/managers_device_global.py

```diff
@@ -12,6 +12,7 @@ class DeviceGlobalCfgMgr(Manager):
     TSA_DEFAULTS = "false"
     WCMP_DEFAULTS = "false"
     IDF_DEFAULTS = "unisolated"
+    RECEIVED_DEFAULTS = "ignore"
```

Comment:
@colintle are u sure these two are equivalent:
```
bgp bestpath bandwidth ignore
no bgp bestpath bandwidth
```
did u test this behaviour on a real/virtual topology?

Reply:
Yeah I tested this in a real topology:

bgp bestpath bandwidth ignore - it will ignore bandwidth from extended community and will only do ECMP

no bgp bestpath bandwidth - it will use W-ECMP if all paths from an advertised prefix have bandwidth. If at least one path does not have bandwidth, it will only use ECMP

Reply:
@colintle so here you changing the default behavior which may affect other customers

Reply:
I made the default to be "ignore" because bandwidth gets cumulated from eBGP peers to eBGP peers, and there is potential that some devices may not want to use bandwidth from advertised routes to calculate weights (W-ECMP) altogether.

Reply:
@colintle i understand the motivation but still believe this should be discussed first. Please involve a wider forum